packages/react/src/components/SkipLink/SkipLink.mdx

```diff
@@ -22,3 +22,35 @@ import { SkipLink } from '@digdir/designsystemet-react';
   Region som kan motta fokus fra skiplink.
 </main>
 ```
+
+## Retningslinjer for `SkipLink`
+
+Noen brukere navigerer med Tab på tastaturet når de skal flytte markøren forbi lenker eller skjemalementer på en nettside. Når vi inkluderer en snarvei-komponent på siden, får disse brukerne muligheten til å gå forbi alle navigeringslenkene øverst på siden og kan gå direkte til hovedinnholdet på siden.
+
+Det er vanligvis et krav at det skal være med en snarvei-komponent i faste sidemaler til alle nettsteder. 
+
+Hvis det er færre enn fem tab-stopp til hovedinnholdet er det ikke sikkert at du trenger å legge inn en `SkipLink`, det fører bare til at det blir flere knapper å forholde seg til.
+
+Vi plasserer snarveien som det første i `<body>`. Hvis siden har et varsel om informasjonskapsler, legger du `SkipLink` rett etter koden for dette varselet.
```

Comment:
Not sure about this sentence. When using react, or other frameworks - the app is not added directly to `body`, but usually a `div`.
This is our storefront page as an example:
![image](https://github.com/digdir/designsystemet/assets/14057205/e5ca3860-84a5-486c-ba92-6b01af727221)

Could we rewrite this to something like:
> Vi plasserer snarveien som det første fokuserbare elementet. Hvis siden har et varsel om informasjonskapsler, legger du `SkipLink` rett etter koden for dette varselet.